src/panels/FlightControlPanel/FlightControlPanel.tsx

```diff
@@ -0,0 +1,141 @@
+import { useEffect } from 'react';
+import { Chip, Container, Group, List, Slider, Switch, Text, Title } from '@mantine/core';
+
+import { useGetBoolPropertyValue, useOpenSpaceApi } from '@/api/hooks';
+import { Tooltip } from '@/components/Tooltip/Tooltip';
+import { setSettings } from '@/redux/flightcontroller/flightControllerSlice';
+import { useAppDispatch, useAppSelector } from '@/redux/hooks';
+import {
+  subscribeToProperty,
+  unsubscribeToProperty
+} from '@/redux/propertytree/properties/propertiesMiddleware';
+import { RollFrictionKey, RotationalFrictionKey, ZoomFrictionKey } from '@/util/keys';
+
+export function FlightControlPanel() {
+  const luaApi = useOpenSpaceApi();
+  const dispatch = useAppDispatch();
+
+  const isControllerEnabled = useAppSelector(
+    (state) => state.flightController.settings.isEnabled
+  );
+  const mouseScaleFactor = useAppSelector(
+    (state) => state.flightController.settings.inputScaleFactor
+  );
+
+  const rotationFriction = useGetBoolPropertyValue(RotationalFrictionKey) ?? false;
+  const zoomFriction = useGetBoolPropertyValue(ZoomFrictionKey) ?? false;
+  const rollFriction = useGetBoolPropertyValue(RollFrictionKey) ?? false;
+
+  useEffect(() => {
+    function subscribeTo(uri: string) {
+      dispatch(subscribeToProperty({ uri }));
+    }
+    function unsubscribeTo(uri: string) {
+      dispatch(unsubscribeToProperty({ uri }));
+    }
+
+    subscribeTo(RotationalFrictionKey);
+    subscribeTo(ZoomFrictionKey);
+    subscribeTo(RollFrictionKey);
+
+    return () => {
+      unsubscribeTo(RotationalFrictionKey);
+      unsubscribeTo(ZoomFrictionKey);
+      unsubscribeTo(RollFrictionKey);
+      dispatch(setSettings({ isEnabled: false }));
+    };
+  }, [dispatch]);
+
+  function toggleEnable() {
+    dispatch(setSettings({ isEnabled: !isControllerEnabled }));
+  }
+
+  function toggleRotation() {
+    luaApi?.setPropertyValueSingle(RotationalFrictionKey, !rotationFriction);
+  }
+
+  function toggleZoom() {
+    luaApi?.setPropertyValueSingle(ZoomFrictionKey, !zoomFriction);
+  }
+
+  function toggleRoll() {
+    luaApi?.setPropertyValueSingle(RollFrictionKey, !rollFriction);
+  }
+
+  const tooltipContent = (
+    <Container>
+      <Text>Interact with the area to control the camera.</Text>
+      <br />
+      <Text fw={'bold'}>Mouse controls:</Text>
+      <Text>Click and drag to rotate. Hold</Text>
+      <List>
+        <List.Item>Shift to zoom (y-axis) or roll (x-axis) </List.Item>
+        <List.Item>Ctrl to pan </List.Item>
+      </List>
+      <br />
+      <Text fw={'bold'}>Touch controls:</Text>
+      <List>
+        <List.Item>1 finger to rotate</List.Item>
+        <List.Item>2 fingers to pan</List.Item>
+        <List.Item>3 fingers to zoom (y-axis) or roll (x-axis)</List.Item>
+      </List>
+    </Container>
+  );
+
+  return (
+    <Container>
+      <Title order={4}>Flight Control Settings</Title>
+      <Group justify={'space-between'}>
```

Comment:
```suggestion
      <Group justify={'space-between'} wrap={'nowrap'}>
```

? Currently the tooltip wraps very uglily when the panel is resized
![image](https://github.com/user-attachments/assets/c23ef438-4c36-4ecf-82c1-af941a596b24)